### PR TITLE
[WIP]: Improve report messages

### PR DESCRIPTION
**Pull request by bocekm**
Related: OAMG-888

### CODE REVIEW

**Comment by vinzenz** [comment]
Need a rebase and merge

**Comment by vojtechsokol** [comment]
leapp-ci build

**Comment by bocekm** [comment]
I still need to improve it a bit.

**Comment by ZhukovGreen** [comment]
closed in favour of https://github.com/oamg/leapp-repository/pull/519

File: repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py [hunk 1]
```diff
@@ -251,13 +251,16 @@ def filter_out_pkgs_in_blacklisted_repos(to_install):
     # FIXME The to_install contains just a limited subset of packages - those that are *not* currently installed and
     # are to be installed. But we should also warn about the packages that *are* installed.
     blacklisted_pkgs = set()
+    blacklisted_repos = get_repositories_blacklisted()
```
Comment (bocekm) [inline repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py:254]: @ZhukovGreen, I've done this change to not call the `get_repositories_blacklisted()` in every pass of the for loop. And there might be a lot of passes.
I know that gcc is able to do such optimization under the hood. I don't think that Python can do that as well.

Comment (ZhukovGreen) [reply repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py:254]: @bocekm Ok, I will consider this;)

Comment (ZhukovGreen) [reply repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/library.py:254]: @bocekm This makes sense. Moreover, in the upstream, this change was added in the beginning of the year by Vinny

**Pull request closed without merging**